### DISCUSSION TITLE
build issue

### DISCUSSION THREAD

**Discussion by ploffredo** [flareact/flareact#220] [General]
[redacted]

**Comment by tj-kev** [comment]
Hi,
do you have a sample repo I can look at?

**Reply by ploffredo** [reply]
Thanks for the reply, yes I have a repo here: https://github.com/pllffrd/flareact-example

I appreciate the help!

**Reply by tj-kev** [reply]
[](https://user-images.githubusercontent.com/15669671/155897769-1fb60661-5eb2-44e1-a8cb-e87887e81845.JPG)Tried your repo and it works fine for me. Not sure why you are getting the error.

I have included my console output and browser screenshot below when running ```npm run dev```.

```
[worker] 🌀  Using namespace for Workers Site "__my-project-workers_sites_assets_preview"
[worker] ✨  Success
[worker] 👂  Listening on http://127.0.0.1:8787
[worker] 🌀  Detected changes...
[worker] 💁  Ignoring stale first change
[worker] [2022-02-27 19:55:19] GET my-project.xxxxxxxxxxxxxxxxx.workers.dev/ HTTP/1.1 200 OK
```

![Capture](https://user-images.githubusercontent.com/15669671/155897836-02e94916-fac7-44e1-b9f3-49301d7f54e0.JPG)


This is from running node v16.14 same as you and a clean ```npm install```. This is also running in WSL which shouldn't make any difference.

**Reply by ploffredo** [reply]
Tried again and everything works great. I do get this notice/alert when running `npm run dev` now but no issues.

Thanks for the help!
 ```
┌─────────────────────────────────────────────────────────────────────────────────┐
[client] │                                                                                 │
[client] │   This version of postcss-preset-env is not optimised to work with PostCSS 8.   │
[client] │                Please update to version 7 of PostCSS Preset Env.                │
[client] │                                                                                 │
[client] │                    If you find issues, you can report it at:                    │
[client] │          https://github.com/csstools/postcss-plugins/issues/new/choose          │
[client] │                                                                                 │
[client] └────────────────────────────────────────────────────────────────────────────────
```
